### ISSUE TITLE
C4AI Command

### ISSUE THREAD

**Issue by AdaptiveStep** [ollama/ollama#3100]
Please add the c4ai-command model 
Its really good at translation and can handle 100 languages.

https://huggingface.co/CohereForAI/c4ai-command-r-v01

**Comment by salah55s** [comment]
Also supporting the quantized versions.. as it's normal is 50+ GB

**Comment by simsi-andy** [comment]
The model has already bern created and it has been implemented (merged) into llama.cpp. How can it be used here? 

https://huggingface.co/andrewcanis/c4ai-command-r-v01-GGUF

https://github.com/ggerganov/llama.cpp/pull/6033

**Comment by Noeda** [comment]
Hey folks. I believe there is a slight issue with tokenization on Command-R on llama.cpp (just opened https://github.com/ggerganov/llama.cpp/issues/6104). I don't think it impacts output quality in a material way but if we've got invested people here on Command-R model maybe you'll just want that issue on your notifications. I plan to investigate it in more detail some time next week.

The tokenization divergence between `llama.cpp` and Command-R Huggingface implementation sometimes seems to slightly re-order top logits depending on how much the tokenization diverges based on your prompt, which can impact what it outputs. But I haven't empirically seen any degradation of quality.

**Comment by mchiang0610** [comment]
Hi! Really sorry for the slow reply to this issue. We've added Command-R model from Cohere to the model library. 

https://ollama.com/library/command-r 

We're working on Command R Plus model right now, and is available in the pre-release build of Ollama. 

Please let us know if you run into any problems! Again, sorry for the slow reply.
